woocommerce/woocommerce-ios
# Edit Product: Enable search categories

Closes: #11379 


## Description

This PR enables search categories for editing products. There's an existing bool check for this so the fix is simple. I updated the UI to move the Add Category button to the bottom of the screen instead of on the top so that the screen looks nicer with the search bar on the top.

Also enabled the Clear selection button and fixed the missing leading margin for the button.

## Testing instructions

- Log in to a store and switch to the Products tab.
- Select an existing product or create a new one.
- On the product form, select Categories (or go to Add more details > Categories).
- The edit category screen should be updated with the search bar on the top and the Add button at the bottom.
- Select any category, the Clear selection button should show up. 
- Tap the Clear selection, all categories should be unselected.

## Screenshots


<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/7ebcf2bd-91d3-4c66-9015-768a5db6d020" width=320 />


---
